test/test_argparsewrapper.py

```diff
@@ -65,9 +68,10 @@ def test_no_action_returns_prints_error(self):
     def test_action_with_unknown_flag_returns_prints_error(self):
         with self.assertRaises(SystemExit):
             with redirect_stderr(self.capture_stream):
-                self.arg_parse.parse_args(["testCommand", "--unknown"])
+                with patch.object(self.arg_parse, '_get_resource_namespace', return_value=None):
+                    self.arg_parse.parse_args(["testCommand", "--unknown"])
 
-        self.assertRegex(self.capture_stream.getvalue(), "Error unrecognized arguments: --unknown")
+        self.assertRegex(self.capture_stream.getvalue(), "Error")
```

Comment:
Why just Error?

Reply:
because it is general enough but added this back in